src/common/misc/FileFunctions.C

```diff
@@ -19,6 +19,11 @@
 #include <userenv.h> // for GetProfilesDirectory
 #include <direct.h>
 #include <Shlwapi.h> // PathIsRelative
+// std::filesystem available in MSVC, it is unclear at the moment which gcc
```

Comment:
IMHO, we should move to new compilers with support for c++17 and beyond sooner rather than later. We are so far behind the curve with some of our third-party libraries, it is becoming difficult to update to newer versions of them when needed.